website/src/theme/SearchBar/styles.module.css

```
@@ -0,0 +1,171 @@
+  justify-content: center;
+  min-height: 44px;
+  min-width: 44px;
+  padding: 0.4rem 0.55rem;
+}
+
+.searchControl:hover,
+.searchControl:focus-visible,
+.closeControl:hover,
+.closeControl:focus-visible {
+  color: var(--ifm-color-primary);
+  outline: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)